src/main/java/wellnus/atomichabit/command/AddCommand.java

```diff
@@ -113,8 +113,7 @@ public void validateCommand(HashMap<String, String> arguments) throws BadCommand
         if (arguments.get(COMMAND_KEYWORD) != "") {
             throw new BadCommandException(AddCommand.COMMAND_INVALID_ARGUMENTS_MESSAGE);
         }
-        String name = arguments.get(AddCommand.COMMAND_NAME_ARGUMENT);
-        if (name.isEmpty()) {
+        if (!arguments.containsKey(AddCommand.COMMAND_NAME_ARGUMENT)) {
```

Comment:
The previous check seems to catch the bug whereby the payload for name is empty, but the new one only checks whether there is name keyword.